tests/unit/test_sds_in_a_box_stack.py

```diff
@@ -4,13 +4,13 @@
 
 import aws_cdk as core
 import aws_cdk.assertions as assertions
-
 from sds_in_a_box.sds_in_a_box_stack import SdsInABoxStack
 
 # example tests. To run these tests, uncomment this file along with the example
 # resource in sds_in_a_box/sds_in_a_box_stack.py
 def test_sqs_queue_created():
-    app = core.App()
+    app = core.App(context={"SDS_ID": "unit-testing"})
+    app_props = core.AppProps(context={"SDS_ID": "unit-testing"})
     stack = SdsInABoxStack(app, "sds-in-a-box")
     template = assertions.Template.from_stack(stack)
```

Comment:
I'm not following this test. Are there assertions in here somewhere? It looks like it is just creating the stack and template, but not guaranteeing anything is created. We also don't have sqs_queue yet?